src/views/clusteroverview/OverviewTab/vms-per-resource-card/VMsPerResourceChart.tsx

```diff
@@ -1,39 +1,47 @@
-import React, { useMemo } from 'react';
+import React, { FC, useMemo } from 'react';
 import { ReactNode } from 'react';
 
 import LoadingEmptyState from '@kubevirt-utils/components/LoadingEmptyState/LoadingEmptyState';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { ChartDonut } from '@patternfly/react-charts';
-import { Card, CardBody, CardHeader, CardTitle, TitleSizes } from '@patternfly/react-core';
+import { CardBody, TitleSizes } from '@patternfly/react-core';
 
-import useVMsPerTemplateResources from './hooks/useVMsPerTemplateResources';
+import useVMsPerResources from './hooks/useVMsPerResources';
 import EmptyStateNoVMs from './utils/EmptyStateNoVMs';
 import RunningVMsChartLegend from './utils/RunningVMsChartLegend';
-import { getChartData, getLegendItems, getTemplateToVMCountMap } from './utils/utils';
+import {
+  changeInstanceTypeName,
+  getChartData,
+  getResourceLegendItems,
+  getResourcesToVMCountMap,
+  vmsPerResourceCount,
+} from './utils/utils';
 
 import './VMsPerTemplateCard.scss';
-
-const VMsPerTemplateCard = () => {
+type VMsPerResourceChartProps = {
+  title: string;
+};
+const VMsPerResourceChart: FC<VMsPerResourceChartProps> = ({ title }) => {
   const { t } = useKubevirtTranslation();
-  const { loaded, vms, templates } = useVMsPerTemplateResources();
+  const { loaded, vms, resources } = useVMsPerResources();
 
-  const templateToVMCountMap = useMemo(
-    () => getTemplateToVMCountMap(loaded, vms, templates),
-    [loaded, vms, templates],
+  const resourceToVMCountMap = useMemo(
+    () => getResourcesToVMCountMap(loaded, vms, resources, title),
+    [loaded, vms, resources, title],
   );
-
-  const chartData = getChartData(templateToVMCountMap);
-  const legendItems = getLegendItems(templateToVMCountMap);
+  const chartData = getChartData(resourceToVMCountMap);
+  const legendItems = getResourceLegendItems(resourceToVMCountMap);
   const numVMs = vms?.length;
+  const vmsPerResourcesCount = vmsPerResourceCount(resourceToVMCountMap);
 
   const RunningVMsChart = (
     <div>
       <ChartDonut
-        ariaDesc={t('VirtualMachines per template')}
-        ariaTitle={t('VirtualMachines per template donut chart')}
+        ariaDesc={t('VirtualMachines per resource')}
+        ariaTitle={t(title)}
```

Comment:
translating a variable doesn't go well with the i18n (it needs to know the exact string it will need to translate). since the strings accepted here are already translated (please make sure it is) you can simply omit the `t` here.